From 5d6b7ad0969c0480e64b0bf8a46acab5fed443b5 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Thu, 10 Mar 2016 11:53:50 +0100
Subject: [PATCH] 2

---
 file_contexts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/file_contexts b/file_contexts
index 84373b8..c423887 100644
--- a/file_contexts
+++ b/file_contexts
@@ -860,3 +860,6 @@
 
 # bash
 /system/xbin/bash                               u:object_r:shell_exec:s0
+
+# Boeffla specific selinux contexts
+/sbin/busybox			u:object_r:toolbox_exec:s0
-- 
1.9.1

