From dea9a164da0599ad25846fe569510e02b5ec6e4a Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Thu, 16 Jun 2016 14:19:08 +0200
Subject: [PATCH] 2

---
 fstab.qcom | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fstab.qcom b/fstab.qcom
index d35314b..c987630 100644
--- a/fstab.qcom
+++ b/fstab.qcom
@@ -18,4 +18,4 @@
 /dev/block/bootdevice/by-name/bluetooth               /bt_firmware      vfat    ro,shortname=lower,uid=1002,gid=3002,dmask=222,fmask=333,context=u:object_r:bt_firmware_file:s0                 wait
 
 /devices/soc.0/f9200000.ssusb/f9200000.dwc3/xhci-hcd* auto              auto    defaults                                                                                                        voldmanaged=usbdisk:auto
-/dev/block/zram0                                      none              swap    defaults                                                                                                        zramsize=536870912,zramstreams=4,notrim
+#AP - no default swap	/dev/block/zram0                                      none              swap    defaults                                                                                                        zramsize=536870912,zramstreams=4,notrim
-- 
1.9.1

